### PR TITLE
fix `compute_entropy`

### PR DESCRIPTION
**Pull request by giovp**

### CODE REVIEW

File: src/moscot/base/problems/_utils.py [hunk 1]
```diff
@@ -721,5 +721,5 @@ def _get_n_cores(n_cores: Optional[int], n_jobs: Optional[int]) -> int:
     return n_cores
 
 
-def _compute_conditional_entropy(p_xy: ArrayLike) -> ArrayLike:
+def _compute_conditional_entropy(p_xy: ArrayLike, c: float = 0.0) -> ArrayLike:
```
Comment (MUCDK) [inline src/moscot/base/problems/_utils.py:724]: We can remove this function then , right?

File: src/moscot/base/problems/_mixins.py [hunk 2]
```diff
@@ -685,12 +685,18 @@ def compute_entropy(
...
+            Constant added to each row of the transport matrix to avoid numerical instability.
+        kwargs
+            Kwargs for :func:`~scipy.stats.entropy`.
 
         Returns
         -------
         :obj:`None` if ``key_added`` is not None. Otherwise, returns a data frame of shape ``(n_cells, 1)`` containing
         the conditional entropy per cell.
         """
+        from scipy import stats
```
Comment (MUCDK) [inline src/moscot/base/problems/_mixins.py:698]: why not import at the top of the script

File: src/moscot/base/problems/_utils.py (hunk 1 above)
Comment (giovp) [reply src/moscot/base/problems/_utils.py]: yes done

File: src/moscot/base/problems/_mixins.py (hunk 2 above)
Comment (giovp) [reply src/moscot/base/problems/_mixins.py:698]: only used by one test, think it's fine to leave it there?

**Comment by giovp** [comment]
@MUCDK update to revert scipy so that it's supported by 3.8. Yet since scipy 1.12 doesn't support 3.8 anymore, I think we should remove it completely from CI eventually

**Review by MUCDK — APPROVED** [review]
LGTM

**Pull request merged**